core/src/css/typography.scss

```diff
@@ -38,6 +38,10 @@ html {
  */
 @supports (-webkit-touch-callout: none) {
   html {
+    /**
+     * Fallback if Dynamic Type is not enabled.
+     */
+    font: 16px var(--ion-font-family);
```

Comment:
I could also change the `font-family` declaration above to use `font: 16px var(--ion-font-family)` instead. Any preference?